js/src/components/RelatedKeyphrasesModal.js

```diff
@@ -15,7 +15,7 @@ import SemRushMaxRelatedKeyphrases from "./modals/SemRushMaxRelatedKeyphrases";
 /**
  * Redux container for the RelatedKeyPhrasesModal modal.
  */
-class RelatedKeyPhrasesModal extends Component {
+class RelatedKeyphrasesModal extends Component {
```

Comment:
If you want to rename this with a lowercase `p` then also all the other occurrences in the documentation etc. should be changed to a lowercase p.